artifacts/deploy/karmada-etcd.yaml

```diff
@@ -82,6 +82,7 @@ spec:
             - --trusted-ca-file=/etc/karmada/pki/etcd-ca.crt
             - --data-dir=/var/lib/etcd
             - --snapshot-count=10000
+            - --cipher-suites=TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256,TLS_AES_256_GCM_SHA384,TLS_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
```

Comment:
It'd be great to have a comment to describe where these suites come from.

Reply:
> It'd be great to have a comment to describe where these suites come from.

+1

Reply:
Referring to [go/src/crypto/tls/cipher_suites.go]( https://github.com/golang/go/blob/b9a08f159d3074ad5921a9d8625b267b64d957bc/src/crypto/tls/cipher_suites.go#L47-L96), golang supports 17 security algorithms.

On this basis, k8s adds map mapping to two of the security algorithms. Finally, these 19 cipher suites were formed.
https://github.com/karmada-io/karmada/blob/98e655fc552b2987c3f2d2a061007889ce8be536/vendor/k8s.io/component-base/cli/flag/ciphersuites_flag.go#L33-L44

Reply:
I mean comment in the code, so that when people read the code they can get where these cipher suites come from.

Reply:
Got it~, I'll comment in the code right away